test/feature/reverse_routing.feature

```diff
@@ -26,7 +27,7 @@ Feature: reverse routing
       GET '/users/:userId/posts/:id', to: 'posts#show', as: 'userPost'
       """
     And an exprestive app using defaults
-    Then I have a routing helper "res.locals.paths.userPost({userId: 1, id: 2})" that returns "/users/1/posts/2"
+    Then I have a routing helper "res.locals.routes.userPost({userId: 1, id: 2})" that returns "/users/1/posts/2"
```

Comment:
That test above is actually not testing what you think it tests and I think I'm going to remove it. Having the routes auto snake-case is a good thing IMO but not implemented yet. Let's do that in a future PR